pkg/util/progressdots/progressdots.go

```diff
@@ -46,14 +46,16 @@ func (s *ProgressDots) Start() {
 			default:
 				fmt.Print(".")
 				time.Sleep(s.interval)
+				s.interval = s.interval + 1*time.Second
```

Comment:
Doesn't this slowdown too quickly?

I would make this optional and over a modulo of time...
This is more important for starting with HyperV than the registration.

Reply:
Yes, with every second it is increasing ...

I think the idea was after some dots(say 10), it should increase per second

Reply:
A certain amount of dots if set... 10 is an arbitrary number.

Else the meaning of interval is obsolete too. In this case it is a startInterval.